### ISSUE TITLE
Windows Chrome Render Issue

### ISSUE THREAD

**Issue by biobronco** [airbnb/lottie-web#1165]
- Created clock in Illustrator
- Imported to After Effects
- Converted to AE shapes
- Animated in AE
- Saved comp with Bodymovin'

Animation works as expected in all **Mac** browsers (Safari, Chrome, FireFox, etc.)

Animation works as expected in most **Windows** browsers, except for Chrome. Windows Chrome does not render the clock correctly.

(No masks or merged paths used in AE.)

<img width="344" alt="clock screenshot 2" src="https://user-images.githubusercontent.com/41762212/43292221-9865ac66-90f1-11e8-959b-8146dfd22aa2.png">

![clock screenshot](https://user-images.githubusercontent.com/41762212/43292222-98831a08-90f1-11e8-8965-9a8c086ea16c.png)

[Clock.aep.zip](https://github.com/airbnb/lottie-web/files/2233784/Clock.aep.zip)

**Comment by bodymovin** [comment]
Hi, I just tested it on window chrome and it seems to work fine.
Can you share a link with your implementation?

**Comment by biobronco** [comment]
Can I DM a link to you with the implementation?

**Comment by bodymovin** [comment]
sure, email me
hernantorrisi[]gmail.com

**Comment by BillyTheGit** [comment]
I had the same problem multiple times, also with a circle drawn in Illustrator.
Although my problem was with Chrome on a iMac, while I had not problem with Safari or Firefox on a iMac and no problem with Firefox, IE, Edge or Chrome on Windows. So the problem existed on Chrome on the iMac. I even tested it in Chrome on a Macbook which gave the same problem, so I was thinking maybe it is a screen/resolution problem?

Than again, I think I solved it by mirroring the image (or maybe I turned it 180 degrees), if that doesn't help I think in one instance I also redraw the circle. So I'm not sure if that can be a screen/resolution thing, if that is even possible.

This was also some versions ago, so I can’t speak for the latest version, but it seems to be some weird bug.

Will let you know more if I run into the same problem again.
